docs/installation-and-operation/information-collection.md

```diff
@@ -1,104 +1,29 @@
+If you've opted in to provide us with anonymous usage data (thank you!), Metabase will phone home some data collected via Google Analytics and Snowplow.
 
-We also will phone home some anonymized metrics from the metabase application server on a nightly basis. Note that we don't collect any usernames, any emails, the server IP, database details of any kind or any personally identifiable information in this process.
+**We only collect Metabase data; we don't collect any of your data.** We don't collect any usernames, emails, server IPs, database details of any kind, or any personally identifiable information.
 
-While this list of anonymous information we collect might seem long, it’s useful to compare this to other alternatives. With a typical SaaS platform, not only will this information be collected, but it will also be accompanied by information about your data, how often it is accessed, the specific queries that you use, specific numbers of records all tied to your company and current plan.
+The data helps us understand how people are actually using Metabase, which in turn helps us prioritize what to work on next.
 
-We collect this information to improve your experience and the quality of Metabase. We use the information you voluntarily share to understand how our users are actually using our product, what kinds of features to prioritize, and how many items to target in our design process. For example, without knowing what the distribution of the number of accounts on each instance in our install base is, we can't know whether the tools we provide can't scale until someone complains. And even then, we only hear complaints and not the people who are quietly happily using us. We're striving to create the best product possible.
+## Opting out of anonymous usage data collection
 
-If you prefer not to provide us with this anonymous usage data, you can always go to your instance’s admin section and toggle off the option for `Anonymous Tracking`.
+To opt out of providing us with your anonymous usage data:
 
-If you're in the process of setting up your Metabase, an admin can also toggle off tracking during the `Usage Data Preferences` onboarding step. We collect a few anonymous events before that point, but won't do so anymore if you choose to opt out.
+1. Click on the gear icon.
+2. Select **Admin settings**.
+3. Go to the **Settings** tab.
+4. Click **General**
+5. Toggle the **Anonymous tracking** option.
 
-In the list below, we spell out exactly why we collect each bit of information.
+If you're in the process of setting up your Metabase, you can also toggle off tracking during the `Usage Data Preferences` onboarding step. We collect a few anonymous events before that point, but won't do so anymore if you choose to opt out.
 
-## Example questions we want to answer
```

Comment:
I think _something_ which explains why we collect and how we use this data seems important. Maybe something which covers:
- We collect events when relevant things happen in the product
- We collect some performance
- We phone home general instance stats like the number and types of DBs your have connected, the number of questions and dashboards in the instance, etc

Using those we're able to answer questions like
- What features are being used vs not
- Where users are getting stuck
- How is performance for key flows (like querying or loading) evolving over time

In a nutshell: assess opportunities for improvement and evaluate the outcome of our projects. It's not for sales or advertising (and it's anonymous, so we couldn't use it for that anyway ;) )